pkg/machines/org.cockpit-project.cockpit-machines.metainfo.xml

```diff
@@ -0,0 +1,17 @@
+<component type="addon">
+  <id>org.cockpit-project.cockpit-machines</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Machines</name>
+  <summary>
+    Manage your virtual machines
+  </summary>
+  <description>
+    <p>
+      This tool manages virtual machines.  With it, you can create,
```

Comment:
"This tools" sounds a bit odd to me. But indeed we have the same verbiage in other metainfos, so consistency :)

Reply:
Yes, I would not object at all to changing this wording (consistently).